pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -631,20 +635,20 @@ func (r *PerformanceProfileReconciler) applyComponents(profile *performancev2.Pe
 
 func (r *PerformanceProfileReconciler) deleteComponents(profile *performancev2.PerformanceProfile) error {
 	tunedName := components.GetComponentName(profile.Name, components.ProfileNamePerformance)
-	if err := r.deleteTuned(tunedName, components.NamespaceNodeTuningOperator); err != nil {
+	if err := resources.DeleteTuned(context.TODO(), r.Client, tunedName, components.NamespaceNodeTuningOperator); err != nil {
```

Comment:
let's create a single context - TODO() is ok and let's reuse alongside the client for all the calls in the function

Reply:
I'm planning on doing it on a later PR. (part 2 of hypershift adoption)

Reply:
ack